### PR TITLE
[disasters] fix bug with leaf pages

### PR DESCRIPTION
**Pull request by beets**
We currently throw an error if there aren't any child place types to fetch (if we are on a leaf page according to the config), introduced in #2448 

Also fixed a couple of strange sounding chart titles

### CODE REVIEW

File: server/routes/sustainability.py [hunk 1]
```diff
@@ -71,8 +71,10 @@ def sustainability_explorer(place_dcid=None):
     if pt in config_place_types:
       place_type = pt
       break
+  contained_place_type = config_place_types[
+      place_type] if place_type != None else None
   subject_config = lib_subject_page_config.remove_empty_charts(
-      subject_config, place_dcid, config_place_types[place_type])
+      subject_config, place_dcid, contained_place_type)
```
Comment (chejennifer) [inline server/routes/sustainability.py:77]: nit: could just do `contained_place_type.get(place_type, None)`?

**Review by chejennifer — APPROVED** [review]
thanks for fixing!

File: server/routes/sustainability.py (hunk 1 above)
Comment (beets) [reply server/routes/sustainability.py:77]: hm, looks like i need both (since place_type could also be None, and .get throws an error then).

**Comment by beets** [comment]
thanks for the review!

**Pull request merged**